src/wrappers/MarkdownWrapper.module.scss

```diff
@@ -26,6 +28,7 @@
 
   // Prism classes, cannot be class modules
   .token.comment,
+  .token.doc-comment.comment.punctuation,
```

Comment:
why did you have to add and move these classes around?

Reply:
Certain token types were not present in the initial scss file and were falling back on the default text color. Adding additional selectors allowed me to make sure nothing was falling through the cracks when theming syntax highlighting.

Reply:
Hold up, I'm having trouble with finding an example to show this right now, so I may have gotten myself turned around at some point. I'll either update with screenshots or remove the class selectors I added once I have more time to investigate.

Reply:
So this is an example of a docstring containing curly brackets without the selector added at line 31.
![Screen Shot 2020-06-15 at 1 33 25 PM](https://user-images.githubusercontent.com/43624800/84693466-221d4280-af0d-11ea-9b0a-d6876ea9b7b1.png)
Especially in a dark theme, it sticks out. After the selector is added, we get this: 
![Screen Shot 2020-06-15 at 1 33 51 PM](https://user-images.githubusercontent.com/43624800/84693531-3eb97a80-af0d-11ea-9efc-0ee493413d93.png)